.github/workflows/ci.yml

```diff
@@ -122,15 +122,17 @@ jobs:
       # TODO: simulate + plot multiple topologies
       - name: Plot mesh grid
         run: |
-          cabal run -- elastic-buffer-sim:sim +RTS -M8M -RTS plot complete3 150000 100
+          cabal run -- elastic-buffer-sim:sim +RTS -M8M -RTS --output-mode pdf complete --nodes 3 --steps 150000 --samples 100
```

Comment:
This still has to be replaced by proper simulation entries. The `sim` executable terminates with an error code now, if the buffers have not stabilized at end of the simulatino. This is why CI still fails at the moment.